app/views/vendors/index.html.erb

```diff
@@ -0,0 +1,110 @@
+<div class="row">
+  <div class="large-4 small-12 columns">
+
+    <img src="http://placehold.it/500x500&text=Logo">
+
+    <div class="hide-for-small panel">
+      <h3>Header</h3>
+      <h5 class="subheader">Risus ligula, aliquam nec fermentum vitae, sollicitudin eget urna. Donec dignissim nibh fermentum odio ornare sagittis.
+      </h5>
+    </div>
+
+    <a href="#">
+    <div class="panel callout radius">
+      <h6>99  items in your cart</h6>
+    </div>
+    </a>
+
+  </div>
+
+
+
+
+
+
+  <div class="large-8 columns">
+    <div class="row">
+
+      <div class="large-4 small-6 columns">
+        <img src="http://placehold.it/1000x1000&text=Thumbnail">
+
+        <div class="panel">
+          <h5>Item Name</h5>
+          <h6 class="subheader">$000.00</h6>
+        </div>
+      </div>
+
+      <div class="large-4 small-6 columns">
+        <img src="http://placehold.it/500x500&text=Thumbnail">
+
+        <div class="panel">
+          <h5>Item Name</h5>
+          <h6 class="subheader">$000.00</h6>
+        </div>
+      </div>
+
+      <div class="large-4 small-6 columns">
+        <img src="http://placehold.it/500x500&text=Thumbnail">
+
+        <div class="panel">
+          <h5>Item Name</h5>
+          <h6 class="subheader">$000.00</h6>
+        </div>
+      </div>
+
+      <div class="large-4 small-6 columns">
+        <img src="http://placehold.it/500x500&text=Thumbnail">
+
+        <div class="panel">
+          <h5>Item Name</h5>
+          <h6 class="subheader">$000.00</h6>
+        </div>
+      </div>
+
+      <div class="large-4 small-6 columns">
+        <img src="http://placehold.it/500x500&text=Thumbnail">
+
+        <div class="panel">
+          <h5>Item Name</h5>
+          <h6 class="subheader">$000.00</h6>
+        </div>
+      </div>
+
+      <div class="large-4 small-6 columns">
+        <img src="http://placehold.it/500x500&text=Thumbnail">
+
+        <div class="panel">
+          <h5>Item Name</h5>
+          <h6 class="subheader">$000.00</h6>
+        </div>
+      </div>
+    </div>
+
+
+
+
+
+    <div class="row">
+      <div class="large-12 columns">
+        <div class="panel">
+          <div class="row">
+
+            <div class="large-2 small-6 columns">
+              <img src="http://placehold.it/300x300&text=Site Owner">
+            </div>
+
+            <div class="large-10 small-6 columns">
+              <strong>This Site Is Managed By<hr/></strong>
+
+              Risus ligula, aliquam nec fermentum vitae, sollicitudin eget urna. Donec dignissim nibh fermentum odio ornare sagittis
+            </div>
+
+          </div>
+        </div>
+      </div>
+
+
+
+    </div>
+  </div>
+</div>
```

Comment:
Is this just the same page as above? Again, okay for now, but promise to change this and add actual content as your very next step.